src/components/workoutsRoadmap/templates/workoutsRoadmapDiagnosticSummary.template.html

```diff
@@ -3,7 +3,7 @@
     <div class="diagnostic-workout-title" translate=".DIAGNOSTIC_TEST"></div>
     <div class="results-text" translate=".DIAG_RES_TEXT"></div>
     <div class="total-score"
-         ng-if="!vm.isSubjectsWaitToBeEvaluated"
+         ng-if="!vm.isSubjectsWaitToBeEvaluated && vm.settings.showCompositeScore"
```

Comment:
This can (should) be a general parameter we can keep in the ENV.
It's useful for the entire app - never use "composite score" because in this app (test) there's no meaning for composite score.

If it's from ENV then we can avoid all of the settings you added?